src/SpectrumCalculationModule.ts

```
@@ -0,0 +1,475 @@
+    return this.highPassFrequency < this.lowPassFrequency;
+  }
+  
+  // Helper to check if integration is allowed
+  private isIntegrationAllowed(): boolean {
+    // High pass filter must be used for integration
+    return this.highPassFrequency > 0;
+  }
+
+  constructor() {
+    // Initialize with default values
+  }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)